cli/script/command-parser.ts

```diff
@@ -599,6 +609,15 @@ function createCommand(): cli.ICommand {
                 }
                 break;
 
+            
+            case "debug":
+                cmd = { type: cli.CommandType.debug };
+
+                var debugCommand = <cli.IDebugCommand>cmd;
```

Comment:
I simply followed the existing pattern used within this file, but I'm actually not sure why we're even creating a new runtime variable simply to appease TypeScript for casting purposes. This seems like a cleaner solution and resolves your comment as well:

```javascript
cmd = <cli.IDebugCommand>{
    type: cli.CommandType.debug,
    platform: arg1
};
```